keylime-agent/src/config.rs

```diff
@@ -828,6 +828,7 @@ mod tests {
     fn get_revocation_notification_ip_empty() {
         let mut test_config = KeylimeConfig {
             agent: AgentConfig {
+                enable_revocation_notifications: true,
```

Comment:
I wonder why this is here ... in as `true`

Reply:
This is to test certain required configuration options when the option is enabled. For that we need to enable the option.